Payfort/Fort/view/frontend/web/EnableMadaLogo.php

```diff
@@ -0,0 +1,50 @@
+<?php
```

Comment:
Move this file and implementation to controller/Payment and follow the structure